src/v3/plugins/github/findReferences.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import * as N from "./nodes";
+import * as R from "./relationalView";
+import {parseReferences} from "./parseReferences";
+
+export type GithubReference = {|
+  src: N.TextContentAddress,
+  dst: N.ReferentAddress,
+|};
+
+type TextContentEntry =
+  | R.IssueEntry
+  | R.PullEntry
+  | R.CommentEntry
+  | R.ReviewEntry;
+export function* findReferences(
+  view: R.RelationalView
+): Iterator<GithubReference> {
+  function* allGithubEntities(): Iterator<R.Entry> {
+    yield* view.repos();
+    yield* view.issues();
+    yield* view.pulls();
+    yield* view.reviews();
+    yield* view.comments();
+    yield* view.userlikes();
+  }
+
+  const refToAddress: Map<string, N.StructuredAddress> = new Map();
```

Comment:
I don’t understand why this is called `refToAddress` and not
`urlToAddress`. It doesn’t map from `GithubReference`s. What
am I missing?